### PR TITLE
Use EMW Filter in Worker Activity Report

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?244563
Behind a toggle.  Needed somewhat urgently for ICDS.  I'll be throwing this up on staging too.
@czue @sravfeyn

### CODE REVIEW

File: corehq/apps/reports/util.py [hunk 1]
```diff
@@ -235,6 +235,16 @@ def parts():
             is_active=user.get('is_active', None)
         )
 
+def get_simplified_users(user_es_query):
```
Comment (dimagimon) [inline corehq/apps/reports/util.py]: E302 expected 2 blank lines, found 1

**Comment by czue** [comment]
note for reviewer in case you get to this before me: please use squash merge so it's easier to hotfix

File: corehq/apps/reports/filters/users.py [hunk 2]
```diff
@@ -295,9 +296,13 @@ def user_es_query(cls, domain, mobile_user_and_group_slugs):
             return q.OR(*user_type_filters)
         else:
             # return matching user types and exact matches
+            location_ids = list(SQLLocation.active_objects
+                                .get_locations_and_children(location_ids)
+                                .location_ids())
             id_filter = filters.OR(
                 filters.term("_id", user_ids),
                 filters.term("__group_ids", group_ids),
+                user_es.location(location_ids),
```
Comment (czue) [inline corehq/apps/reports/filters/users.py]: does this change the behavior of this filter for all reports that use it? and if so, is that desired?

**Review by czue — APPROVED** [review]
looks good to me - just one clarifying question

File: corehq/apps/reports/filters/users.py (hunk 2 above)
Comment (czue) [inline corehq/apps/reports/filters/users.py]: offlined with @benrudolph and we convinced ourselves that this is better behavior.

**Comment by benrudolph** [comment]
also noting this should fix this as well: http://manage.dimagi.com/default.asp?244877#1267846

**Comment by sravfeyn** [comment]
Should this be tagged with `To be patched on NIC` or is there a change in patch process?

**Comment by benrudolph** [comment]
already patched

**Comment by sravfeyn** [comment]
Sweet!

File: corehq/apps/reports/filters/users.py (hunk 2 above)
Comment (esoergel) [inline corehq/apps/reports/filters/users.py]: :+1:

**Pull request merged**